docs/content/test/TestMinimized.md

```diff
@@ -50,9 +50,11 @@ If this is your case, add these resources to `res/raw/keep.xml` like this:
            tools:keep="@layout/page_object*"/>
 ```
 
-### Try latest stable version of R8
+### X already has a mapping
 
-Some issues probably solved in new version of r8. For example: ["already has a mapping" one](https://issuetracker.google.com/issues/122924648)
+[Issue](https://issuetracker.google.com/issues/122924648)
+
+Solved in a new version of r8.
```

Comment:
which one? Without exact version, it's hard to check

Reply:
i don't know, it's kinda hard to find

Reply:
it is probably 1.6.x, because reproduced in 1.5.x